packages/components/src/Banner/Banner.test.tsx

```diff
@@ -29,6 +29,17 @@ it("renders a warning banner", () => {
   expect(tree).toMatchSnapshot();
 });
 
+it("renders without close button", () => {
+  const tree = renderer
+    .create(
+      <Banner type="warning" dismissible={false}>
+        Warn
+      </Banner>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
Can we achieve this test without a snapshot? would something like below be a more suitable option?

```
const { queryByLabelText } = render(<Banner dismissible={false}>Foo</Banner>
expect(queryByLabelText("Close")).toBeNull()
```

Reply:
@eddysims Updated the test and also removed the snapshot.  Thanks!